src/interface/Type.ts

```diff
@@ -0,0 +1,22 @@
+export interface LinkType {
+  id: number;
+  created_at: string;
```

Comment:
c: 생성일과 같이 string으로 나타내는 date 값의 경우 어떤 포맷인지 타입이나 주석으로 나타내는 것이 좋습니다.

```suggestion
  /**
   * ISO 8601
   * @example 2023-06-04T14:38:28.719907+00:00
   */
  created_at: string;
  // 혹은 created_ap: ISO8601Date;
```